Onboarding/sbm-send-pregnancy-message.md

```diff
@@ -88,20 +105,81 @@ card GetMessage when date == today(), then: SendMessage do
   page = page.body
 end
 
+card GetMessage when schedule_date != today(), then: CalculateTimestamp do
+  page = page + 1
+end
+
 card GetMessage, then: CalculateTimestamp do
   page = page + 1
 end
 
-card SendMessage when page.body.is_whatsapp_template do
-  send_message_template("@lower(page.body.whatsapp_template_name)", "en_US", ["@contact.name"],
+card SendMessage when page.body.is_whatsapp_template, then: CheckContentSetMaxID do
```

Comment:
Have we checked that we actually go to `CheckContentSetMaxID`? I'm not sure of this, but I would expect `send_message_template` to work similarly to `ask` or `buttons`, where it waits for user input, and then goes to the target card specified by the buttons, and only goes to the `then` state if the user does something other than pressing a button. Is that not the case here?

Reply:
Will have a look at this